app/controllers/ontologies_controller.rb

```diff
@@ -13,9 +13,6 @@ class OntologiesController < InheritedResources::Base
 
   respond_to :html, except: %i(show)
 
-  before_filter :check_write_permission, except: [:index, :show, :oops_state]
-  before_filter :check_read_permissions
```

Comment:
This is odd. It should not have happened. I probably messed something up with my editor without noticing it. This change is removed (by commit amend) in 318de55.